inc/consent/namespace.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Altis\Core\Consent;
```

Comment:
We just use the `Altis` namespace for core stuff, `Altis\Core` isn't used anywhere.
```suggestion
namespace Altis\Consent;
```

Reply:
I added the `Core` to distinguish it from `Altis\Consent` because that would introduce a function name clash with `Altis\Consent\bootstrap`

Reply:
I added `Altis\Core\` because there is already an [`Altis\Consent\bootstrap()`](https://github.com/humanmade/altis-consent/blob/master/inc/namespace.php#L18) which would clash with [`Altis\Core\Consent\bootstrap()`](https://github.com/humanmade/altis-core/blob/12b0c1aedd2d1492af48a399955cd9decc3f239c/inc/consent/namespace.php#L11) if left in the `Altis\Consent` namespace. 

I wasn't sure what the best way to handle this clash was other than creating a `Altis\Core` namespace. I considered renaming the function `init()`, but I was afraid of other potential clashes with `Altis\Consent` if I left the `Core` module using the same namespace.

Reply:
Ah I see, this seems like a reasonable workaround then 👍